Shipfinity.Api/appsettings.json

```diff
@@ -7,7 +7,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "ShipfinityDbString": "Data Source=localhost\\SQLEXPRESS;Database=ShipfinityDb;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"
+    "ShipfinityDbString": "Data Source=(localdb)\\LocalDBDemo;Database=ShipfinityDbTest;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"
```

Comment:
revert changes to connection string to `"Data Source=localhost\\SQLEXPRESS;Database=ShipfinityDb;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"`

Reply:
done